notebook/agentchat_mongodb_RetrieveChat.ipynb

```diff
@@ -0,0 +1,1510 @@
+{
```

Comment:
Have you ran the notebook? @ranfysvalle02 
It didn't work for me.

Reply:
@thinkall how are you trying to run it? Can you please share the error? I will look into this today

Reply:
I think I found the issue!

`TypeError: MongoDBAtlasVectorDB.create_collection() missing 2 required positional arguments: 'index_name' and 'similarity'`

Reply:
Hi @ranfysvalle02 ,  the dependency is not correct:
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], [line 4](vscode-notebook-cell:?execution_count=1&line=4)
      [1](vscode-notebook-cell:?execution_count=1&line=1) import json
      [2](vscode-notebook-cell:?execution_count=1&line=2) import os
----> [4](vscode-notebook-cell:?execution_count=1&line=4) from sentence_transformers import SentenceTransformer
      [6](vscode-notebook-cell:?execution_count=1&line=6) import autogen
      [7](vscode-notebook-cell:?execution_count=1&line=7) from autogen.agentchat.contrib.retrieve_assistant_agent import RetrieveAssistantAgent

ModuleNotFoundError: No module named 'sentence_transformers'
```

Reply:
@thinkall - I made some updates to the implementation, and made the notebook a bit more simple. Can you please give it a try now and let me know what errors you run into? 

I grabbed the example notebook from `# Using RetrieveChat for Retrieve Augmented Code Generation and Question Answering`, copy pasted it, modified it for MongoDB Atlas. It doesn't "error out", but the output is not what I expect from looking at the other notebooks. Any ideas? 

Here is what I've tried so far:
- Programmatic Time Delay to make sure index is "ready"

We are definitely "close" ..

<img width="840" alt="Screenshot 2024-06-21 at 7 50 26 AM" src="https://github.com/microsoft/autogen/assets/6230773/9d6ae1f8-ca3f-4f51-8aab-6e66800e108d">

Reply:
Why removed the docs for launch mongodb with docker-compose? After setting mongodb with the docker-compose file in your last version,  now  run with the current version, I  get below error:
```
ConfigurationError: The DNS query name does not exist: _mongodb._tcp.shared.demo.mongodb.net.
```

Reply:
@ranfysvalle02 , the notebook should work with a new env, in which, we only do `pip install -e ".[retrievechat-mongodb]"`. Of course,  we'll  also update the config_list.

Reply:
"Why removed the docs for launch mongodb with docker-compose? After setting mongodb with the docker-compose file in your last version, now run with the current version, I get below error:"

This was a mistake -- standby... I think we are getting closer

Reply:
@thinkall - Can you give it a try now? I think we're almost there!